shell/app/styles/antd-extension.scss

```diff
@@ -326,10 +326,6 @@ input.ant-input {
   display: inline-flex;
 }
 
-.api-filter-input {
-  width: auto;
-}
```

Comment:
will this deletion affect other place?

Reply:
> will this deletion affect other place?

![image](https://user-images.githubusercontent.com/30014895/125619799-8814153e-93f9-48dd-b610-ae2c3039fbea.png)
only one place use the className